src/chaplin/views/view.coffee

```diff
@@ -194,7 +194,7 @@ module.exports = class View extends Backbone.View
   #   e.g.
   #   @delegate('click', 'button.confirm', @confirm)
   delegate: (eventName, second, third) ->
-    if Backbone.utils
+    if Backbone.utils and Backbone.utils?.delegate
```

Comment:
Mind telling me what they are? We could maybe do a 0.6.4.

You might also want to try just using regular old Backbone. Why do you need Exoskeleton?